.gitignore

```diff
@@ -128,5 +128,7 @@ dmypy.json
 # Pyre type checker
 .pyre/
 
+# important/cached data non required on github
 /sadcowpng/sadcowpng/settings.py
 .idea/
+sadcowpng/sharedspaces/migrations/
```

Comment:
Apologies if I missed a discussion on this, but shouldn't we commit `/migrations` so that our database schema is up to date with our merges? I understand we may have to do some coordination, but I think we should consider removing this from the .gitignore.

Reply:
Yep, thank you for pointing it out.